sql/2019/07_Performance/07_20.sql

```diff
@@ -0,0 +1,31 @@
+#standardSQL
+# 07_20: Percentiles of scripting CPU time
+#corresponding to the time main thread of the browser was busy
+SELECT
+  client,
+  ROUND(APPROX_QUANTILES(scriptingCpuTime, 1000)[OFFSET(100)] / 1000, 2) AS p10,
+  ROUND(APPROX_QUANTILES(scriptingCpuTime, 1000)[OFFSET(250)] / 1000, 2) AS p25,
+  ROUND(APPROX_QUANTILES(scriptingCpuTime, 1000)[OFFSET(500)] / 1000, 2) AS p50,
+  ROUND(APPROX_QUANTILES(scriptingCpuTime, 1000)[OFFSET(750)] / 1000, 2) AS p75,
+  ROUND(APPROX_QUANTILES(scriptingCpuTime, 1000)[OFFSET(900)] / 1000, 2) AS p90
+FROM 
+( 
+  SELECT 
+  _TABLE_SUFFIX AS client,
+  (
+    CAST(IFNULL(JSON_EXTRACT(payload, "$['_cpu.EvaluateScript']"), "0") AS INT64) +

```

Comment:
@pmeenan sanity check: are these the right CPU metrics to sum to calculate total scripting time?

Reply:
Ping